boards/arm/verdin_imx8mm_m4/verdin_imx8mm_m4_defconfig

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2023, Toradex
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_SOC_SERIES_IMX8MM_M4=y
+CONFIG_SOC_MIMX8MM6=y
+CONFIG_BOARD_VERDIN_IMX8MM_M4=y
+CONFIG_CLOCK_CONTROL=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
```

Comment:
Is this Kconfig required as the "minimal boot set" of Kconfigs? IE does the UART driver need to be set to interrupt mode in order to function, or can the board boot without this Kconfig set

Reply:
Removed, thanks